hd/etc/copyr.txt

```diff
@@ -72,11 +72,13 @@
         </div>
         <div class="col-9">
           <div class="btn-group float-right">
-            <span class="align-self-center">%version; - %commit; (%commit_date;)
-              %if;(bvar.display_compilation_time="on")([compiled on] %compilation_time;)%end;%sp;
-            &copy; <a href="https://www.inria.fr/">INRIA</a> 1998-2017 %connections; %sq;</span>
             <a href="https://github.com/geneweb/geneweb" class="ml-1" target="_blank" title="GeneWeb sources on GitHub">
             <img src="%image_prefix;/logo_bas.png" alt="Logo GeneWeb"/></a>
+            <span class="align-self-center">
+              &nbsp;&nbsp;V&nbsp;<a href="https://github.com/geneweb/geneweb/commit/%commit;" 
```

Comment:
You can you Bootstrap class `ml-1` or `ml-2` (**m**argin **l**eft) instead of `%nbsp;%nbsp;` to add margin before text.